python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -1336,18 +1336,27 @@ def test_torch_tensor_nccl_comm_deduplicate_collectives(ray_start_regular):
     compiled_dag = dag.experimental_compile()
 
     from ray.dag.collective_node import CollectiveOutputNode
+    from ray.experimental.channel import ChannelContext
 
-    comms = set()
+    comm_ids = set()
     for _, exec_tasks in compiled_dag.actor_to_executable_tasks.items():
         for exec_task in exec_tasks:
             dag_node = compiled_dag.idx_to_task[exec_task.task_idx].dag_node
             if isinstance(dag_node, CollectiveOutputNode):
                 assert exec_task.collective_group is not None
                 nccl_group_id = exec_task.collective_group._type_hint._nccl_group_id
                 assert nccl_group_id is not None
-                comms.add(nccl_group_id)
+                comm_ids.add(nccl_group_id)
     # Only 1 NCCL group should be created.
-    assert len(comms) == 1
+    assert len(comm_ids) == 1
+    comm_id = list(comm_ids)[0]
+    ctx = ChannelContext.get_current()
+    comm = ctx.nccl_groups[comm_id]
+    comm_actors = comm.get_actor_handles()
+    # The NCCL group should contain both workers.
+    assert set(comm_actors) == set(workers)
+    # p2p NCCL group should be None b/c no tensors are sent via NCCL.
```

Comment:
Change to "P2P NCCL group should be None since NCCL transport is not used.".

Reply:
done